.goreleaser.develop.yaml

```diff
@@ -174,11 +174,12 @@ docker_signs:
   - artifacts: all
     stdin: "{{ .Env.COSIGN_PASSWORD }}"
 
-archives:
-  - files:
-      - src: tmp/{{ .Os }}_{{ .Arch }}/libs/*
-        dst: libs
-        strip_parent: true
+archives: []
+  # - rlcp: true
```

Comment:
Needs a comment here too to explain why this is commented and whey/when we would uncomment if we want to leave it

Reply:
added